### PR TITLE
chore(weave): Unblock Python unit tests canceling 

### PR DESCRIPTION
**Pull request by vwrj**

### CODE REVIEW

File: weave/tests/test_arrow_vectorizer.py [hunk 1]
```diff
@@ -1008,41 +1008,42 @@ def test_arrow_dict():
...
-                        "_base_type": {"type": "list"},
-                        "objectType": "int",
-                    },
-                    "varName": "x",
-                },
-                "other": {"nodeType": "const", "type": "int", "val": 1},
-            },
-        },
-    }
+# TODO: Temporarily commenting out this test to prevent Python unit tests from timing out and canceling
```
Comment (jlzhao27) [inline weave/tests/test_arrow_vectorizer.py:1011]: nit, isn't there a `@test.skip` or something to make this a little cleaner?

Comment (tssweeney) [inline weave/tests/test_arrow_vectorizer.py:1011]: nit: much cleaner to use a `pytest.mark.skip`

**Review by tssweeney — APPROVED** [review]

File: weave/tests/test_arrow_vectorizer.py (hunk 1 above)
Comment (vwrj) [reply weave/tests/test_arrow_vectorizer.py]: done

Comment (vwrj) [reply weave/tests/test_arrow_vectorizer.py]: done

**Pull request closed without merging**